### ISSUE TITLE
unable to uninstall

### ISSUE THREAD

**Issue by xatabhk** [aspnet/Tooling#601]
It says uninstalling is successfully, but all VS2015 Tooling Preview builds are always shown in the list.
![cannot be uninstalled](https://cloud.githubusercontent.com/assets/6814395/16219091/49575610-37b6-11e6-9906-9b4c0fcca22a.png)

**Comment by balachir** [comment]
@xatabhk did you uninstall and install the tooling builds, or do you do build-to-build upgrade? Can you zip up and share the setup logs - you can find them in your %temp% folder with file names beginning with "dd_DotNetCoreTools"? Thanks!

**Comment by xatabhk** [comment]
@balachir No. I just installed the new versions. Here the log files:
[Temp_log.zip](https://github.com/aspnet/Tooling/files/330721/Temp_log.zip)

**Comment by BillHiebert** [comment]
@joeloff

**Comment by joeloff** [comment]
Looks like it reported a prompt for source error when trying to remove the previous bundle:

[2604:088C][2016-06-21T14:12:08]e000: Error 0x80070643: Failed while prompting for source (original path 'C:\Users\zaxideler\Downloads\DotNetCore.1.0.0.RC2-VS2015Tools.Preview1.exe').
[2604:088C][2016-06-21T14:12:08]e311: Failed to acquire container: WixAttachedContainer to working path: C:\Users\ZAXIDE~1\AppData\Local\Temp\{2F1FB010-6C13-4BAB-9501-CFBCCE3CDF99}\9E533866687847BA46211D2976C4A65B32A4B06E, error: 0x80070643.
[2604:3678][2016-06-21T14:12:08]e000: Error 0x80070643: Cache thread exited unexpectedly.

And again for one of the nightly preview builds

[2B90:1D74][2016-06-21T14:12:09]i371: Updating session, registration key: SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{7afd06bf-3d0f-4522-8211-7bccc5d76dd8}, resume: Active, restart initiated: No, disable resume: Yes
[29DC:3870][2016-06-21T14:12:09]w341: Prompt for source of container: WixAttachedContainer, path: C:\Users\zaxideler\Downloads\DotNetCore.1.0.0.RC3-27-VS2015Tools.Preview2.exe
[29DC:3870][2016-06-21T14:12:09]e054: Failed to resolve source for file: C:\Users\zaxideler\Downloads\DotNetCore.1.0.0.RC3-27-VS2015Tools.Preview2.exe, error: 0x80070643.
[29DC:3870][2016-06-21T14:12:09]e000: Error 0x80070643: Failed while prompting for source (original path 'C:\Users\zaxideler\Downloads\DotNetCore.1.0.0.RC3-27-VS2015Tools.Preview2.exe').
[29DC:3870][2016-06-21T14:12:09]e311: Failed to acquire container: WixAttachedContainer to working path: C:\Users\ZAXIDE~1\AppData\Local\Temp\{998EF230-B991-4DC6-9201-2022737E0434}\5A0EDFCEC21851418BC109FCD76DB9951FB0E53F, error: 0x80070643.
[29DC:33C8][2016-06-21T14:12:09]e000: Error 0x80070643: Cache thread exited unexpectedly.

**Comment by xatabhk** [comment]
@joeloff Yes. There was dialog, but just for build of Nightly 6/7. The other two builds can be uninstalled 'successfully'.

**Comment by barrytang** [comment]
TFS Bug#235762

**Comment by balachir** [comment]
@xatabhk, if you still have the issue with multiple entries in ARP, you can try to workaround the issue by downloading the installers for '[Nightly 6/17](https://download.microsoft.com/download/B/D/1/BD13D9A4-71FC-4C9B-B67B-38540773B978/DotNetCore.1.0.0.RC3-34-VS2015Tools.Preview2.exe)' and '[Preview 1](https://download.microsoft.com/download/4/6/1/46116DFF-29F9-4FF8-94BF-F9BE05BE263B/DotNetCore.1.0.0.RC2-VS2015Tools.Preview1.exe)', then try to run those uninstall again. Does that solve the problem?

**Comment by xatabhk** [comment]
@balachir Tried already; it did no work.
